decksite/views/rotation_changes.py

```diff
@@ -0,0 +1,14 @@
+from decksite.view import View
+
+# pylint: disable=no-self-use
+class RotationChanges(View):
+    def __init__(self, cards_in, cards_out):
+        self.sections = []
+        self.cards = cards_in + cards_out
+        entries_in = [{'n': None, 'name': c.name, 'card': c} for c in cards_in]
+        entries_out = [{'n': None, 'name': c.name, 'card': c} for c in cards_out]
+        self.sections.append({'name': 'New to the format', 'entries': entries_in, 'num_entries': len(entries_in)})
```

Comment:
Nit. I'd probably use 'New to the Format' and 'Left the Format'.

Reply:
I was pondering if it should say New this season. No idea of the equivalent for leaving.

Reply:
'New this Season' and 'Rotated Out'? What do they call it on dailymtg.com?